package.json

```diff
@@ -3,7 +3,7 @@
   "author": "Microsoft Application Insights Team",
   "license": "MIT",
   "bugs": "https://github.com/microsoft/ApplicationInsights-node.js/issues",
-  "version": "3.0.0-beta.11",
+  "version": "3.0.0-beta.12",
```

Comment:
Are you not updating the dependencies? are these updated already?

Reply:
They're updated now